ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-api-models.j2

```diff
@@ -0,0 +1,26 @@
+{# Management API Models #}
+{% if management_api_models is arista.avd.defined %}
+
+## Management API Models
+
+### Management API Models Summary
+
+| Provider | Path | Enabled |
+| -------- | ---- | ------- |
+{%     if management_api_models.providers is arista.avd.defined %}
+{%         for provider in management_api_models.providers | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
+{%             if provider.paths is arista.avd.defined %}
```

Comment:
Since the keys supported will depend on the provider type, I think we should implement the check now, so it will be apparent to the next contributor how to add another provider type.
```suggestion
{%             if provider.name is arista.avd.defined and provider.name in ['smash', 'sysdb'] %}
```

Reply:
👍 will add check for supported providers